packages/list-plugins/lib/tracking/byTracking.js

```diff
@@ -0,0 +1,70 @@
+const { Relationship } = require('@keystone-alpha/fields');
+
+const composeResolveInput = (originalHook, newHook) => async params => {
```

Comment:
Let's make composeResolveInput a utility function and import it into both of these rather than write the same function twice.

Reply:
moved to util.js